src/main/java/core/basesyntax/service/operations/BalanceOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
```

Comment:
Don't begin class or method implementation with empty line. Remove the redundant empty line at the start of the class definition.